btrack/_localization.py

```diff
@@ -188,16 +192,17 @@ def segmentation_to_objects(
     # objects
     if "label" in properties:
         logger.warning("Cannot use scikit-image label as a property.")
-        del properties["label"]
+        properties = set(properties)
+        properties.remove("label")
+        properties = tuple(properties)
```

Comment:
This seems like an odd combination of steps. So `properties` comes in as a `tuple` (which are immutable), then change to a mutable `set` to remove something, and then convert back to `tuple`. Is there any benefit of `properties` being a `tuple` originally? Why not make it a `set`?

Reply:
I agree that it's a bit weird.  The idea was to have a common interface with scikit image `regionprops` which uses a tuple as input. However, `label` is used as a property in `regionprops` and internally in btrack to mean something else.  Was trying to strike a balance here. Happy to change it though